test/src/utils.rs

```diff
@@ -100,3 +111,27 @@ where
 pub fn clear_messages(net: &Net) {
     while let Ok(_) = net.receive_timeout(Duration::new(0, 100)) {}
 }
+
+pub fn since_from_relative_block_number(block_number: BlockNumber) -> u64 {
+    FLAG_SINCE_RELATIVE | FLAG_SINCE_BLOCK_NUMBER | block_number
+}
+
+pub fn since_from_absolute_block_number(block_number: BlockNumber) -> u64 {
+    FLAG_SINCE_BLOCK_NUMBER | block_number
+}
```

Comment:
I think those functions should add to `enum SinceMetric` and should use in `send_transaction` RPC in the future.